src/js/plugins/plugin.chapters.js

```diff
@@ -42,50 +57,70 @@ BookReader.prototype.init = (function(super_) {
 BookReader.prototype.addChapter = function(chapterTitle, pageNumber, pageIndex) {
   const uiStringPage = 'Page'; // i18n
   const percentThrough = BookReader.util.cssPercentage(pageIndex, this.getNumLeafs() - 1);
-  const jumpToChapter = (event) => this.jumpToIndex($(event.target).data('pageIndex'));
-  const title = `Chapter: ${chapterTitle} | ${uiStringPage} ${pageNumber}`;
-
-  $(`<div><div>${title}</div></div>`)
-    .addClass('BRchapter')
-    .css({ left: percentThrough })
-    .appendTo(this.$('.BRnavline'))
-    .data({ pageIndex })
-    .bt({
-      contentSelector: '$(this).find("> div")',
-      trigger: 'hover',
-      closeWhenOthersOpen: true,
-      cssStyles: {
-        padding: '12px 14px',
-        backgroundColor: '#fff',
-        border: '4px solid rgb(216,216,216)',
-        color: 'rgb(52,52,52)'
-      },
-      shrinkToFit: false,
-      width: '230px',
-      padding: 0,
-      spikeGirth: 0,
-      spikeLength: 0,
-      overlap: '0px',
-      overlay: false,
-      killTitle: false,
-      offsetParent: null,
-      positions: ['top'],
-      fill: 'white',
-      windowMargin: 10,
-      strokeWidth: 0,
-      cornerRadius: 0,
-      centerPointX: 0,
-      centerPointY: 0,
-      shadow: false
-    })
-    .hover(event => {
+  const jumpToChapter = (event) => {
+    this.jumpToIndex($(event.target).data('pageIndex'));
+    $('.current-chapter').removeClass('current-chapter');
+    $(event.target).addClass('current-chapter');
+  }
+  const title = `${chapterTitle} | ${uiStringPage} ${pageNumber}`;
+
+  //adding items to mobile table of contents
+  const mobileChapter = $(`<li>${title}</li>`);
```

Comment:
This will prevent XSS injection, since the data on OL is user-entered.

```suggestion
  const mobileChapter = $(`<li></li>`).text(title);
```

Reply:
done